source/installguide/hypervisor/xenserver.rst

```diff
@@ -31,12 +31,17 @@ System Requirements for XenServer Hosts
    See the Citrix Hardware Compatibility Guide:
    `http://hcl.xensource.com <http://hcl.xensource.com>`_
 
-    -  XenServer 5.6 SP2
-    -  XenServer 6.0
-    -  XenServer 6.0.2
-    -  XenServer 6.1.0
-    -  XenServer 6.2.0
-    -  XenServer 6.5.0
+    -  XenServer 7.0 
+    -  XenServer 7.1
+    -  XenServer 7.5
+    -  XenServer 8.0
+    -  XenServer 8.1
+    -  XCP-ng 7.4
+    -  XCP-ng 7.5
+    -  XCP-ng 7.6
+    -  XCP-ng 8.0
+    -  XCP-ng 8.1
```

Comment:
Missing 8.2 - also, should we add `.0` on each of them? As per @weizhouapache's comment on other PR: https://github.com/apache/cloudstack-documentation/pull/272#discussion_r863680841